memgpt/cli/cli_config.py

```diff
@@ -95,9 +95,9 @@ def configure():
     if use_openai or use_azure:
         model_options = []
         if use_openai:
-            model_options += ["gpt-3.5-turbo", "gpt-3.5", "gpt-4"]
+            model_options += ["gpt-3.5-turbo", "gpt-4", "gpt-4-1106-preview"]
         default_model = questionary.select(
-            "Select default model (recommended: gpt-4):", choices=["gpt-3.5-turbo", "gpt-3.5", "gpt-4"], default="gpt-4"
+            "Select default model (recommended: gpt-4):", choices=["gpt-3.5-turbo", "gpt-4", "gpt-4-1106-preview"], default="gpt-4"
```

Comment:
I made the diff here too because I just grep'd for `gpt-3.5-turbo`, but I'm not sure if this is needed too.